data/themes/default/style.css

```diff
@@ -551,9 +551,9 @@ BBTCOView {
 
 /* Plugins */
 
-TripleOscillatorView Knob {
-	color: rgb(1, 32, 64);
-	qproperty-outerColor: rgb(0, 0, 0);
+TripleOscillatorView knob {
```

Comment:
Yup, it seems like `knob` should be `Knob` and that's what broke it.

@LocoMatt can you please fix this?